src/classes/RD2_ScheduleService_TEST.cls

```diff
@@ -1373,8 +1373,105 @@ private with sharing class RD2_ScheduleService_TEST {
 
     }
 
-
-        //// Helpers
+//    /****
```

Comment:
Is there a reason this is commented out?

Reply:
Active schedule handling has been moved to RD2_VisualizeScheduleController.  Tests will be rewritten and moved as well.  Commenting is temporary.